app/templates/cli.js

```diff
@@ -1,9 +1,13 @@
 #!/usr/bin/env node
-'use strict';
+'use strict';<% if (babel) { %>
+import meow from 'meow';
+import <%= camelModuleName %> from './';
+
+const cli = meow([<% } else { %>
 var meow = require('meow');
 var <%= camelModuleName %> = require('./');
 
-var cli = meow([
+var cli = meow([<% } %>
```

Comment:
Not sure if I understand you correctly, can you elaborate? Sindre is using it in every CLI he makes with support for Node 4+. Example: [active-win-cli](https://github.com/sindresorhus/active-win-cli/blob/master/cli.js) and many others.